tests/playbooks/test_tripleo_adoption_requirements.yaml

```diff
@@ -0,0 +1,43 @@
+- name: TripleO adoption requirements
+  hosts: "{{ groups['overcloud'][0] | default([]) }}"
+  gather_facts: true
+  tasks:
+    # Propagate StorageNFS network
+    - name: Propagate Storage NFS network
```

Comment:
maybe I can put a condition to skip this in case we want to execute against a  standalone environment.

Reply:
Done